apps/component-date/README.md

```diff
@@ -0,0 +1,3 @@
+Date component example
+============
+
```

Comment:
Please add it now. We should consider "go and look at the code" as a failure of documentation, even if we do have nicely commented examples. The code here should be a reference for a finished version, but the README should tell the user how they would implement in their own app.

This also helps to make it clear to novice users what parts of the code example are relevant to the functionality in question. Otherwise they find themselves copying random unrelated things in an effort to "make it look like the example".